test/channelRepository.test.js

```diff
@@ -0,0 +1,77 @@
+const expect = require('chai').expect;
+const channelRepository = require('../src/repositories/channelRepository');
+
+describe('channelRepository', () => {
+    describe('adding new channel to empty repository', () => {
+        it('should make repository to have elements', async () => {
+            await channelRepository.createChannel('foo');
+            const channelList = await channelRepository.channelList();
+            expect(channelList.length).to.greaterThan(0);
+        });
+    });
+
+    describe('adding twice a channel to repository', () => {
+        it('should avoid duplicated channel', async () => {
+            await channelRepository.createChannel('comics');
+            try {
+                await channelRepository.createChannel('comics');
+            } catch (error) {
+                expect(error).equals('Channel comics already exist.');
+            }
+        });
+    });
+
+    describe('adding twice a user to a channel repository', () => {
+        it('should avoid duplicated user', async () => {
+            await channelRepository.addUser('comics', 'jai');
+            try {
+                await channelRepository.addUser('comics', 'jai');
+            } catch (error) {
+                expect(error).equals('jai already exists in this channel.');
+            }
+        });
+    });
+
+    describe('request messages from channel with messages', () => {
+        it('should return messages', async () => {
+            const message = {
+                channelKey: 'comics',
+                userId: 'jai',
+                text: 'batman is awesome',
+            };
+            await channelRepository.addMessage(message);
+            const messages = await channelRepository.getMessages(message.channelKey);
+            expect(messages.length).to.greaterThan(0);
+        });
+    });
```

Comment:
Would it be a good idea to create tests for empty string, and maybe userId or ChannelKey null or undefined?